nfd-worker.conf.example

```diff
@@ -1,3 +1,4 @@
+# All changes in this template should be applied to Helm chart too.
```

Comment:
Please remove this line. It's not needed as `values.yaml` is update by `make templates`